examples/cim/gnn/actor.py

```diff
@@ -0,0 +1,369 @@
+import ctypes
+from collections import defaultdict, OrderedDict
+import multiprocessing
+from multiprocessing import Process, Pipe, Event, Manager
+import numpy as np
+import time, os, pickle
+
+import torch
+
+from maro.rl import AbsActor
+from maro.simulator import Env
+from maro.simulator.scenarios.cim.common import Action, DecisionEvent
+from maro.utils import Logger, LogFormat, convert_dottable
+from examples.cim.gnn.actor_critic import ActorCritic
+from examples.cim.gnn.action_shaper import DiscreteActionShaper
+from examples.cim.gnn.utils import fix_seed, gnn_union
+from examples.cim.gnn.experience_shaper import ExperienceShaper
+from examples.cim.gnn.state_shaper import GNNStateShaper
+from examples.cim.gnn.shared_structure import SharedStructure
+from examples.cim.gnn.utils import decision_cnt_analysis, compute_v2p_degree_matrix
+
+
+def organize_exp_list(experience_collections:dict, idx_mapping:dict):
+    """The function assemble the experience from multiple processes into a dictionary.
+
+    Args:
+         experience_collections (dict): It stores the experience in all agents. The structure is the same as what is 
+            defined in the SharedStructure in the ParallelActor except additional key for experience length. For 
+            example:
+
+            {
```

Comment:
According to the discussion with Arthur, no need to refine this kind of case in docstring now (since it's under examples).